lib/shimmer/utils/localizable.rb

```diff
@@ -45,7 +45,7 @@ def translate(key, options = {})
             option_names = options.except(:count, :default, :raise, :scope).map { |k, v| "#{k}=#{v}" }.join(", ")
             return "#{key_name} #{option_names}" if I18n.debug && !untranslated
 
-            options.reverse_merge!(default: old_translate(key, **options.merge(locale: :de))) if untranslated
+            options.reverse_merge!(default: old_translate(key, **options.merge(locale: default_locale))) if untranslated
```

Comment:
almost. actually, you don't even need the whole line anymore, this is just a copy/paste residue from the recordsale times where we have `key` as a locale to activate the debugging behavior.

Reply:
Not sure I got it, but removed the line altogether.